test/methods.js

```diff
@@ -470,6 +470,42 @@ asyncTest("remote extensions", function() {
 	strictEqual( v.element(e), true, "still invalid, because remote validation must block until it returns; dependency-mismatch considered as valid though" );
 });
 
+test("remote, data previous querystring", function() {
+	expect(4);
+	var change = true,
+		e = $("#username"),
+		v = $("#userForm").validate({
+			rules: {
+				username: {
+					required: true,
+					remote: {
+						url: "users.php",
+						type: "POST",
+						data: {
+							email: function() {
+								if ( change ) {
+									change = false;
+									return "email.com";
+								}
+								return "email2.com";
+							}
+						},
+						beforeSend: function(request, settings) {
+							if ( change ) {
+								deepEqual(settings.data, "username=asdf&email=email.com");
```

Comment:
could you point me to the test you are referring to? I quickly scanned the method.js and couldn't find where this is the case.